src/main/config/config.c

```diff
@@ -452,7 +452,7 @@ static void validateAndFixConfig(void)
         motorConfigMutable()->dev.useContinuousUpdate = false;
     }
 
-#if defined(USE_DSHOT_TELEMETRY)
+#if defined(USE_DSHOT_TELEMETRY) && defined(USE_TIMER)
```

Comment:
IMO most of code silently assumes that timer is available. testing it explicitly will affect a lot of code

Reply:
Yes it's a bit messy. I'm focussed on ./src/main anything in STM / platform can stay silently using timer.